.github/workflows/markdownlint.yml

```
@@ -0,0 +1,21 @@
+name: Markdownlint
+
+on: pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Run Markdownlint
+      run: |
+        echo "::add-matcher::.github/workflows/markdownlint-problem-matcher.json"
+        npm i -g markdownlint-cli
+        markdownlint "entity-framework/core/**/*.md"
```

I'd probably expand out the glob to `"**/*/md"` unless there is a reason to scope it down

---

entity-framework/ef6 documentation was ported from MSDN in past. We don't actively work on that documentation anymore (the product is also not actively developed either). It is better to exclude those docs here.
Include entity-framework/efcore-and-ef6 here please.

---

OK, it would probably be like this then, to cover everything but the old folder
```suggestion
        markdownlint "**/*.md" -i "entity-framework/ef6/"
```